src/components/widgets/index.js

```diff
@@ -1,18 +1,18 @@
-import HomerWidget from './homer/HomerWidget.vue';
+import HomerWidget from "./homer/HomerWidget.vue";
```

Comment:
@mjhea0 is it prettier making the decision to move from single-quotes to double-quotes?  this isn't something we've completely had a decision on like we've done for Python (double-quotes on Python is Eldarion/Pinax settled style), but i've been using single-quotes on all js.  I'm don't really have strong feelings either way and if the defaults here in the lint/prettier are enforcing double-quotes, I'm fine to roll with it.  If it's just as easy to use single-quotes, that does seem more common in js.  I don't know.  Again, no strong feelings but do want us to settle it here and make everything else consistent one way or another.

Reply:
Yes, prettier was making this decision. I updated the prettier config to allow the use of single quotes.